### PR TITLE
fix: #186 - Passcode Documentation

### LINKED ISSUE #186: Document the files in mifopay/passcode package
**Summary:**

Document the files in mifopay/passcode package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode

### PR DESCRIPTION
**Pull request by theamritanair**
Fixes #186 
- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by theamritanair** [comment]
@luckyman20  please review this PR.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/PassCodeContract.java [hunk 1]
```diff
@@ -4,13 +4,21 @@
...
 
 /**
- * Created by ankur on 15/May/2018
+ * A contract class working as a Interface between the UI and Presenter of Passcode
+ * @author ankur
+ * @since 15/May/2018
  */
 public interface PassCodeContract {
+    /**
+    * Contains all the functions of the UI component
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/PassCodeContract.java]: Align this properly.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java [hunk 2]
```diff
@@ -49,11 +51,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
     }
 
+    /**
+     * This function return the logo
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java]:
```suggestion
     * This function returns the logo
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java [hunk 3]
```diff
@@ -65,22 +74,39 @@ public void startNextActivity() {
         startActivity(intent);
     }
 
+
+    /**
+    * This function starts the LoginActivity
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java]: Align this properly

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java [hunk 4]
```diff
@@ -65,22 +74,39 @@ public void startNextActivity() {
...
+     */
     @Override
     public void startLoginActivity() {
         Intent intent = new Intent(PassCodeActivity.this, LoginActivity.class);
         startActivity(intent);
     }
 
+    /**
+    * This function displays the current context in a Toast.
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java]: Align this too.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java [hunk 5]
```diff
@@ -65,22 +74,39 @@ public void startNextActivity() {
...
+     * @param view The cuurent view
+     * @param msg A message of type integer.
+     */
     @Override
     public void showToaster(View view, int msg) {
         Toast.makeText(getApplicationContext(), getText(msg), Toast.LENGTH_SHORT).show();
     }
 
+    /**
+    * This function return the Encryption tyoe.
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java]: Align this too.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java [hunk 6]
```diff
@@ -65,22 +74,39 @@ public void startNextActivity() {
...
+     */
     @Override
     public int getEncryptionType() {
         return EncryptionUtil.DEFAULT;
     }
 
+
+    /**
+    * This function sets the Presenter.
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/passcode/ui/PassCodeActivity.java]: Align this too.

**Pull request closed without merging**